yaml/overlays/dev/calicodev.yaml

```diff
@@ -284,6 +290,12 @@ spec:
                 configMapKeyRef:
                   name: calico-config
                   key: CALICOVPP_TAP_RING_SIZE
+            - name: CALICOVPP_TAP_MTU
```

Comment:
You can remove this since we will use the host's original MTU for the vpp-manager tap

Reply:
As discussed offline, let's actually keep this as an option to override the default computed MTU for vpptap0.